backend/spotihue/tasks.py

```diff
@@ -11,21 +11,65 @@
 logger = logging.getLogger(__name__)
 
 
+def is_spotihue_running() -> bool:
+    """Queries for the SpotiHue task by its cached spotihue_task_id. There should only be one
+    run_spotihue task running at any given time.
+
+    Returns:
+         bool: Whether the SpotiHue (run_spotihue) task is running.
+    """
+    celery_inspect = celery_app.control.inspect()
+    spotihue_task_id = redis_client.get(constants.REDIS_SPOTIHUE_TASK_ID)
+
+    if spotihue_task_id:
+        spotihue_task_id = spotihue_task_id.decode('utf-8')
+
+        task_query_result = celery_inspect.query_task(*[spotihue_task_id, ])
```

Comment:
Oh, maybe ignore my previous comment because you check that the task is running here?